### PR TITLE
*: move ResetToVersion to debugpb

### PR DESCRIPTION
**Pull request by longfangsong**
Signed-off-by: longfangsong <longfangsong@icloud.com>

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

**Review by lhy1024 — COMMENTED** [review]
so much file diff？

File: proto/debugpb.proto [hunk 1]
```diff
@@ -259,3 +263,10 @@ message GetAllRegionsInStoreRequest {
 message GetAllRegionsInStoreResponse {
     repeated uint64 regions = 1;
 }
+
+message ResetToVersionRequest {
+    uint64 ts = 1;
+}
+
+message ResetToVersionResponse {
+}
```
Comment (disksing) [inline proto/debugpb.proto:272]: Pleaase add an empty line here.

**Comment by longfangsong** [comment]
> so much file diff？

I chaned `kvrpcpb` and IMO all protos which import it need to regenerate go file when `make go`.

File: proto/debugpb.proto (hunk 1 above)
Comment (longfangsong) [reply proto/debugpb.proto]: > Do we need a resp head here?

In fact, there're no infomation we can return here.

**Review by disksing — APPROVED** [review]

**Pull request merged**